packages/expo-cli/src/credentials/provider/iOSCredentialsProvider.ts

```diff
@@ -15,16 +15,21 @@ export interface iOSCredentials {
   };
 }
 
+interface Options extends AppLookupParams {
+  nonInteractive: boolean;
+  skipCredentialsCheck: boolean;
+}
+
 export default class iOSCredentialsProvider implements CredentialsProvider {
   public readonly platform = 'ios';
   private readonly ctx = new Context();
   private credentials?: iOSCredentials;
 
-  constructor(private projectDir: string, private app: AppLookupParams) {}
+  constructor(private projectDir: string, private app: Options) {}
```

Comment:
I would prefer not to merge those values, pass 3 arguments to constructor instead